org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.xtend

```diff
@@ -69,7 +69,7 @@ class ParentProjectDescriptor extends ProjectDescriptor {
 	}
 	
 	def String getTychoVersion() {
-		'1.2.0-SNAPSHOT'
+		'1.2.0'
```

Comment:
Could we only use 1.2.0 when Java 10 is selected, otherwise 1.1.0?
And then leave the comment out?

Reply:
well ill have a look when i have 5 hours time to adjust the tests. we need a better way to adjust them